UnityProjectDP/Assets/Scripts/Visualization/Animation/Animation.cs

```diff
@@ -492,10 +493,10 @@ public void HighlightObject(long objectUniqueId, bool isToBeHighlighted)
         public void HighlightMethod(string className, string methodName, bool isToBeHighlighted, long instanceId = -1)
```

Comment:
```suggestion
    public void HighlightMethod(string className, string methodName, bool isToBeHighlighted, long instanceId = -1)
        {
            var node = classDiagram.FindNode(className);
            if (node)
            {
                ClassTextHighligter classTextHighligter = node.GetComponent<ClassTextHighligter>();
                if (classTextHighligter)
                {
                    if (isToBeHighlighted)
                    {
                        classTextHighligter.HighlightClassLine(methodName);
                    }
                    else
                    {
                        classTextHighligter.UnhighlightClassLine(methodName);
                    }
                }
                else
                {
                    Debug.Log("TextHighlighter component not found");
                }
            }
            else
            {
                Debug.Log("Node " + className + " not found");
            }
        }
```

ale ajtak sa mi to neajko nepaci :D